vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboard.html

```diff
@@ -57,7 +90,36 @@ <h4>Modules</h4>
 				<br><a class="add" th:href="@{/staff/module/add}"><span class="icon-circle-add" style="margin: auto; color: var(--med-grey)"></span></a>
 			</div>
 			</div>
-    </div>      
+    </div> 
+    <div id="spaceLinkAlert" class="modal" tabindex="-1" role="dialog"
+                backdrop="static"
+                style="display: none; background-color: rgba(0, 0, 0, 0.5);">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h5 class="modal-title">Login</h5>
+                        </div>
+                        <div class="modal-body">
+                            <h6>There are space links with source space as Null. Do you want to delete them?</h6>
```

Comment:
should be: 
There are space links with no source space. Do you want to delete them?

Reply:
Done